ci/bazel-scripts/diff.sh

```diff
@@ -44,7 +44,7 @@ if grep -qE ".*\.hs" <<<"$DIFF_FILES"; then
 fi
 
 if grep -qE ".*\.proto" <<<"$DIFF_FILES"; then
-    files+=(//pre-commit:protobuf-format-check)
+    files+=(//pre-commit:protobuf-format-check //pre-commit:buf-breaking)
```

Comment:
I think this will work, but might be a little clearer to add it to a new line like:
```
files+=(//pre-commit:buf-breaking)
```

Reply:
Good point about having the targets on newlines. Although I don't like to repeat myself so what about what I just pushed:
```sh
    files+=(
        //pre-commit:protobuf-format-check
        //pre-commit:buf-breaking
    )
```